src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -172,7 +172,9 @@ const ProgressiveProfiling = (props) => {
   const handleSkip = (e) => {
     e.preventDefault();
     window.history.replaceState(props.location.state, null, '');
-    setShowModal(true);
+    if (welcomePageContext.nextUrl !== '') {
+      setShowModal(true);
+    }
```

Comment:
Why are we changing the functionality?

Reply:
Modal shows the button, click to redirect to the dashboard but in the embedded experience we want to redirect to the next URL.

Reply:
Then lets change the text of the button. We don't want to change functionality without asking product and UX

Reply:
Actually, I didn't set the env variables locally that's why I was getting the label button on the modal button "Go to dashboard". On stage and prod the modal button label is "Go to edX" which is fine. We don't need to add this check.